docs/modules/ROOT/pages/how-tos/upgrade.adoc

```diff
@@ -1,16 +1,59 @@
 = How to Upgrade K8up
 
+WARNING: Never remove the CRDs, as you might lose the respective resources!
+
+[#upgrade_1_to_2]
+== Upgrade K8up from 1.x to 2.x
+
+TIP: See also xref:explanations/what-has-changed-in-v2.adoc[].
+
+Only little has changed on from a user's perspective in K8up 2.x, but a lot did change under the hood nonetheless.
+Some of these changes may require you to update your configuration of K8up.
```

Comment:
With the CRD rename AND version bump from v1alpha1 to v1 in #509 , I'm not sure we can call it "only little has changed"